### ISSUE TITLE
Public sector's property/facility management data

### ISSUE THREAD

**Issue by tormi** [okestonia/Estonian-Open-Data-Issue-Tracker#47]
Public sector's property/facility management data is currently very fragmented or even missing. Let's describe what we have and how to improve the availability of that data in an open format.

- State Real Property Register, https://github.com/okestonia/Estonian-Open-Data-Issue-Tracker/issues/21
- Tartu, https://github.com/okestonia/Estonian-Open-Data-Issue-Tracker/issues/41#issuecomment-286766614

**Comment by tormi** [comment]
- RKAS, http://rkas.ee/kinnisvaraportfell#0

**Comment by boamaod** [comment]
I'm not sure, whether it should be only public sector... The data might be mostly held by public authorities, but this can be data about property belonging to private entities too. Or is this already separate issue?

**Comment by tormi** [comment]
You're correct, I removed that restriction.

**Comment by tormi** [comment]
* City of Tallinn, https://kinnisvara.tallinn.ee/. Data is not (yet?) available in http://avaandmed.tallinn.ee/

**Comment by andreskytt** [comment]
EHR should go on this list, right?

**Comment by tormi** [comment]
No, let's keep EHR separately. EHR doesn't deal with FM and usage contracts.

**Comment by infokujur** [comment]
@tormi is there any change after launching portal of Tallinn with geospacial data? https://opendata.riik.ee/andmehulgad/tallinna-geoportaal/
